frontend/src/services/HttpRequest.js

```diff
@@ -13,7 +15,19 @@ const baseUrl = "http://localhost:5000"
  * @param jsonBody A JS object that will be the JSON body of the request. E.g, {title: "New Project"}
  * @returns {Promise<{body: any, status: number}>}
  */
-async function makeHttpRequest(endpoint, method = "GET", jsonBody = {}) {
+async function MakeHttpRequest(endpoint, method = "GET", jsonBody = {}) {
```

Comment:
I think you accidentally changed the capitalization of the first letter, it should be `makeHttpRequest` since it's a function